src/librbd/ImageCtx.h

```diff
@@ -317,7 +317,9 @@ namespace librbd {
     journal::Policy *get_journal_policy() const;
     void set_journal_policy(journal::Policy *policy);
 
-    static ThreadPool *get_thread_pool_instance(CephContext *cct);
+    static void get_thread_pool_instance(CephContext *cct,
+                                         ThreadPool **thread_pool,
+                                         ContextWQ **op_work_queue);
```

Comment:
@dillaman Looking at get_thread_pool_instance usage, why not crate a function that just returns work queue?